libmscore/layoutlinear.cpp

```diff
@@ -162,6 +162,7 @@ void Score::resetSystems(bool /*layoutAll*/, LayoutContext& lc)
                         }
                   else if (m->header())
                         m->removeSystemHeader();
+                  m->removeSystemTrailer();
```

Comment:
Would it make sense to only do this if `m->trailer()` returns `true`?

Reply:
Yes, caught that and fixed it moments later :-)